.github/workflows/codeql.yml

```diff
@@ -5,11 +5,8 @@
 name: "Code Scanning"
 
 on:
-  push:
-    branches: [main]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [main]
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
If I read this right, this will run the code scanning every day?

Reply:
Yeah, i wanted to change it to run daily instead of running on PRs, since its taking so long